btb/tuning/gcp.py

```diff
@@ -0,0 +1,285 @@
+from __future__ import division, print_function
+from builtins import zip, range
+
+import numpy as np
+import scipy.stats as st
+from scipy.stats import norm
+
+from btb.tuning import Tuner, Uniform
+from sklearn.gaussian_process import GaussianProcess, GaussianProcessRegressor
+
+
```

Comment:
It might be nice to add a comment marking the start and end of the helper functions
`#################Helper Functions#################
#################End Helper Functions#################`